core/services/synchronization/telemetry_ingress_batch_worker.go

```diff
@@ -6,23 +6,23 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
-	"github.com/smartcontractkit/chainlink/v2/core/services"
 	telemPb "github.com/smartcontractkit/chainlink/v2/core/services/synchronization/telem"
-	"github.com/smartcontractkit/chainlink/v2/core/utils"
 )
 
 // telemetryIngressBatchWorker pushes telemetry in batches to the ingress server via wsrpc.
 // A worker is created per ContractID.
 type telemetryIngressBatchWorker struct {
-	services.ServiceCtx
+	services.Service
```

Comment:
services.Service is alias for services.ServiceCtx. Why replace it just here?

Reply:
I swapped the package too, so it's `chainlink-common/pkg/services.Service` now. This is what `core/services.ServiceCtx` is already aliased to.

Reply:
Yeah, I am just asking why change it here to refer directly to chainlink-common/pkg/services.Service and not in some other places in the codebase that are in the same situation

Reply:
These came up because the package imports collide otherwise. We should switch them all and deprecate the old one for consistency, and the EVM & common stuff will have to before it can be moved.